go.mod

```diff
@@ -1,22 +1,80 @@
 module code.cloudfoundry.org/eirinix
 
+replace (
+	github.com/coreos/bbolt => github.com/coreos/bbolt v1.3.0
+	google.golang.org/grpc => google.golang.org/grpc v1.29.1
```

Comment:
just as a reminder: https://github.com/golang/go/issues/30354

modules depending on won't respect the replace directive, so for e.g. grpc here is affected

Reply:
Sigh... I know that. Without those replace statements the code does not compile :( We have the very same replace lines in Eirini as well.
I wish people (I am looking at you, Google) bumped their dependencies more regularly!